embark-consult.el

```diff
@@ -240,7 +285,7 @@ actual type."
 
 (define-key embark-become-file+buffer-map "Cb" #'consult-buffer)
 
-;;; support for Consult search commands
+;;; Support for Consult search commands
```

Comment:
I guess I can go change all the headings in embark.el to start with an uppercase too. 😆

Reply:
Sorry, I saw one in uppercase and changed the rest too, but I think it got out of hand :-P

Reply:
Now I capitalized the one you missed in `embark-consult.el` and all the headings in `embark.el`. 😛 cb43a991ff86e1ae3db4592753b62787b19f3e35